tests/installation/installer_desktopselection.pm

```diff
@@ -44,7 +44,9 @@ sub run() {
     assert_screen "$d-selected";
     if (check_var('VERSION', 'Tumbleweed')) {
         send_key 'alt-o';                                                      # configure online repos
+        wait_still_screen 3;                                                   # wait for the potential 'low memory warning' to show up
         assert_screen 'repo-list';
+        wait_screen_change { send_key $cmd{ok} } if match_has_tag 'repo-list-low_memory_warning';
```

Comment:
logically I would expect us to first wait for the low mem warning before asserting on the repo-list screen to be correctly shown